com.syntevo.SmartGit.json

```diff
@@ -0,0 +1,57 @@
+{
+  "app-id": "com.syntevo.SmartGit",
+  "command": "smartgit",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "separate-locales": false,
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=host",
+    "--share=ipc",
+    "--share=network",
+    "--socket=ssh-auth",
+    "--socket=wayland",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [
+    {
+      "name": "smartgit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -dm0755 /app/bin/",
+        "install -dm0755 /app/smartgit/",
+        "tar -xzvf smartgit-linux*.tar.gz --directory /app/smartgit/ --strip-components=1",
+        "rm smartgit-linux*.tar.gz",
+        "install -Dm0755 smartgit.sh /app/bin/smartgit",
+        "install -Dm0644 /app/smartgit/bin/smartgit-64.png /app/share/icons/hicolor/64x64/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 /app/smartgit/bin/smartgit-128.png /app/share/icons/hicolor/128x128/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 /app/smartgit/bin/smartgit-256.png /app/share/icons/hicolor/256x256/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 -t /app/share/appdata/ com.syntevo.SmartGit.appdata.xml",
+        "install -Dm0644 -t /app/share/applications/ com.syntevo.SmartGit.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
```

Comment:
should be `type: extra-data`

Reply:
Are there any really advantages using `extra-data`? I noticed that if using `extra-data` I have to copy all the images I want to support in the repository now (and every time they change); with `file` I just reference them within `build-commands`.

I prefer how it is currently, but if there is a strong fundamental reason to use `extra-data` I can go for it; I have both approaches working anyway.

Reply:
Yes, following US Copyright law is the advantage. We don't have the right to distribute this software so users must download it at runtime which is how `extra-data` works.

Reply:
Ah! Didn't know about that particular detail :policeman:

I'll push the changes using `extra-data` then...and will do the same for `SmartSynchronize`.